codespell_lib/data/dictionary.txt

```diff
@@ -14415,6 +14539,7 @@ content-negotitation->content-negotiation
 content-negotition->content-negotiation
 content-negoziation->content-negotiation
 contentended->contended
+contentin->contenting, content in, contention,
```

Comment:
Perhaps because I happen to live in France, and remember the Normandy landings on D-Day:
```suggestion
contentin->contenting, content in, contention, Cotentin,
```